src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -0,0 +1,977 @@
+import enum
+import inspect
+import logging
+import subprocess
+from collections import defaultdict
+from math import ceil
+from typing import Any, Mapping, Optional, Sequence, Tuple, Union
+
+import numpy as np
+import tiledbvcf
+
+import tiledb.cloud
```

Comment:
To avoid circular import weirdness, it would be best to import the specific modules that contain whatever you're using.